indico/htdocs/js/indico/modules/tracks.js

```diff
@@ -0,0 +1,62 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2016 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+(function(global) {
+    'use strict';
+
+    global.setupTrackManagement = function setupTrackManagement() {
+        var heightLimit = 50;
+        $('#track-list').on('indico:htmlUpdated', function() {
+            $('.track-content').each(function() {
+                var $this = $(this);
+                if ($this.height() > heightLimit) {
+                    $this.addClass('transparent-overlay track-content-collapsed track-content-collapsible');
```

Comment:
`transparent-overlay` always gets added/removed with `track-content-collapsed`. I propose completely omitting `trasparent-overlay` here and add it directly to `track-content-collapsed`. It would look like that in the SCSS file.

```scss
.track-content-collapsed {
    @extend .transparent-overlay;
    max-height: 50px;
    overflow: hidden;
}
```